components/fxa-client/src/commands/send_tab.rs

```diff
@@ -40,16 +42,31 @@ impl EncryptedSendTabPayload {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SendTabPayload {
     pub entries: Vec<TabHistoryEntry>,
+    // XXX - tests don't fail yet even though none add these 2 new items and
+    // they aren't marked as optional for serde, which seems odd!
+    #[serde(rename = "flowID")]
+    pub flow_id: String,
+    #[serde(rename = "streamID")]
+    pub stream_id: String,
 }
 
 impl SendTabPayload {
-    pub fn single_tab(title: &str, url: &str) -> Self {
-        SendTabPayload {
-            entries: vec![TabHistoryEntry {
-                title: title.to_string(),
-                url: url.to_string(),
-            }],
-        }
+    pub fn single_tab(title: &str, url: &str) -> (Self, SentReceivedTab) {
```

Comment:
From a readability pespective, I wonder if it would be better to refer to this new struct by qualified name, like `telemetry::SentReceivedTab`. This might make it more obvious to the reader what the extra return value is for.